Core/include/Acts/Propagator/StandardAborters.hpp

```diff
@@ -181,6 +181,57 @@ struct EndOfWorldReached {
   }
 };
 
+/// This is the condition that the end of world has been reached
+/// it then triggers an propagation abort
+struct VolumeConstraintAborter {
+  /// boolean operator for abort condition without using the result
+  ///
+  /// @tparam propagator_state_t Type of the propagator state
+  /// @tparam navigator_t Type of the navigator
+  ///
+  /// @param [in,out] state The propagation state object
+  /// @param [in] navigator The navigator object
+  /// @param logger a logger instance
+  template <typename propagator_state_t, typename stepper_t,
+            typename navigator_t>
+  bool checkAbort(propagator_state_t& state, const stepper_t& /*stepper*/,
+                  const navigator_t& navigator, const Logger& logger) const {
+    const auto& constrainToVolumeIds = state.options.constrainToVolumeIds;
+    const auto& endOfWorldVolumeIds = state.options.endOfWorldVolumeIds;
+
```

Comment:
I think in this case the user should not add the aborter to the list to avoid the overhead. I expect that the list is not empty if the aborter is added. Maybe we have to reconsider this if the aborter is added by default but right now it is only implicitly added by the user

Reply:
For the CKF we actually run it all the time so maybe you are right. Does not hurt